### PR TITLE
[ZH/EN] Add osu! FumoFumo Cup tournament articles

### PR DESCRIPTION
**Pull request by CloneWith**
Both of the two languages include basic information, staff table, mappool layout and match results. Detailed rule lays in the Simplified Chinese one.

Date formats for translations need to be checked briefly.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+  - cirno
+  - "OFFC: cirno"
+  - OFFC
+---
+
+# osu! FumoFumo Cup: cirno
+
+![OFFC: cirno logo](img/logo.png)
+
+The **osu! FumoFumo Cup: cirno** (***OFFC: cirno***) is a 4v4 double-elimination osu!standard tournament hosted by ::{ flag=CN }:: [kitakore](https://osu.ppy.sh/users/33887163), which is also the inaugural season of OFFC.
```
Comment (Nivalyx) [inline wiki/Tournaments/OFFC/cirno/en.md:13]:
```suggestion
The **osu! FumoFumo Cup: cirno** (***OFFC: cirno***) is a 4v4 double-elimination osu!standard tournament hosted by ::{ flag=CN }:: [kitakore](https://osu.ppy.sh/users/33887163). It is the inaugural season of the osu! FumoFumo Cup tournament series.
```

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | One MeowPad for each team member (provided by *DESU.Life*) |
+| Top three teams | Corresponding KanonBot badge for each team member |
+| All players | Title in OFFC official QQ group, animated profile banners (team customized) |
+
+Apart from prizes above, the following players chosen by members in the OFFC official QQ group would also be awarded with:
```
Comment (Nivalyx) [inline wiki/Tournaments/OFFC/cirno/en.md:34]:
```suggestion
Apart from the prizes above, the following players as chosen by members in the OFFC official QQ group will also be awarded with:
```

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+| Quarterfinals | 2024-09-09/2024-09-17 |
+| Semifinals | 2024-09-18/2024-09-30 |
+| Finals | 2024-10-01/2024-10-07 |
+| Grand finals | 2024-10-08/2024-10-20 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | One MeowPad for each team member (provided by *DESU.Life*) |
```
Comment (Nivalyx) [inline wiki/Tournaments/OFFC/cirno/en.md:30]: might want to add a link to MeowPad here, as it's not really clear to the readers what a "MeowPad" is on the first glance

something like this :

image

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+| Commentator | ::{ flag=CN }:: [_eilphy](https://osu.ppy.sh/users/15175276), ::{ flag=CN }:: [5atori Komeiji](https://osu.ppy.sh/users/34282059), ::{ flag=CN }:: [Akari Date](https://osu.ppy.sh/users/32701801), ::{ flag=CN }:: [Amrzs0971](https://osu.ppy.sh/users/30973609), ::{ flag=CN }:: [Dr_Shindo](https://osu.ppy.sh/users/30895664), ::{ flag=CN }:: [eric1388](https://osu.ppy.sh/users/20384257), ::{ flag=CN }:: [FroZZZZZen](https://osu.ppy.sh/users/29235972), ::{ flag=CN }:: [kitakore](https://osu.ppy.sh/users/33887163), ::{ flag=HK }:: [lelxdme](https://osu.ppy.sh/users/13795588), ::{ flag=CN }:: [My Angel Kuroe](https://osu.ppy.sh/users/13217389), ::{ flag=CN }:: [Naughty_chas](https://osu.ppy.sh/users/15349720), ::{ flag=CN }:: [Nosenso](https://osu.ppy.sh/users/30329943), ::{ flag=HK }:: [[SHK] Arcadia](https://osu.ppy.sh/users/13888511), ::{ flag=HK }:: [stllok](https://osu.ppy.sh/users/14817468) |
+| Showcase Replay Provider | ::{ flag=CN }:: [Ledeau_Fox](https://osu.ppy.sh/users/15816872) |
```
Comment (Nivalyx) [inline wiki/Tournaments/OFFC/cirno/en.md:58]:
```suggestion
| Showcase replay provider | ::{ flag=CN }:: [Ledeau_Fox](https://osu.ppy.sh/users/15816872) |
```

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+| Team 1 | Team 2 | Match link |
+| --: | :-- | :-- |
+| **这是什么** | Bingo! | [#5](https://osu.ppy.sh/community/matches/115660580) |
+| Pingu~Pingu~ | **沟槽的烤翅还在开战舰追我** | [#6](https://osu.ppy.sh/community/matches/115676257) |
+| 身为pp刷子因没练图而成了传说 | **诟撡𤷙𣧏𡣀𠔑𡉄𫟺㻵𧂂𨾻𠪧** | [#7](https://osu.ppy.sh/community/matches/115662073) |
+| **Helicopter** | 内有小南梁 | [#8](https://osu.ppy.sh/community/matches/115576295) |
+
+## Ruleset
+
+For more information about the rules of **OFFC: cirno**, please turn to our Simplified Chinese wiki page.
```
Comment (Nivalyx) [inline wiki/Tournaments/OFFC/cirno/en.md:225]:
```suggestion
For more information about the rules of **OFFC: cirno**, please refer to our Simplified Chinese wiki page.
```

File: wiki/Tournaments/OFFC/cirno/img/teamup_formula.svg
Comment (Joehuu) [inline wiki/Tournaments/OFFC/cirno/img/teamup_formula.svg:1]: I don't think wiki supports svg. Should be png.

File: wiki/Tournaments/OFFC/en.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# osu! FumoFumo Cup
+
+Index page for all of the osu! FumoFumo Cup series.
+
+- [osu! FumoFumo Cup cirno](cirno)
```
Comment (Joehuu) [inline wiki/Tournaments/OFFC/en.md:5]: Should have the colon as that is shown as the heading of the main tournament article:
```suggestion
- [osu! FumoFumo Cup: cirno](cirno)
```

File: wiki/Tournaments/de.md [hunk 7]
```diff
@@ -1,3 +1,8 @@
+---
+outdated_translation: true
+outdated_since: b75cab0564e5b20a69771d98bbd25edc426b3dad
```
Comment (Joehuu) [inline wiki/Tournaments/de.md:3]: Tournament index translations don't need to be outdated as you can copy the table heading and date formatting of other tournaments.

File: wiki/Tournaments/en.md [hunk 8]
```diff
@@ -295,6 +295,12 @@ Unofficial tournaments/competitions hosted by the communities.
 | osu! European Tournament 3 | 2019-02-08 | 2020-03-23 | Woopak :CRAB: | Fondermanp2p | Käptn Mein Käptn |
 | [osu! European Tournament 2021](o!ET/2021) | 2021-05-15 | 2021-07-11 | opa pizdec | The Cow Fields | Team Papayapa |
 
+#### osu! FumoFumo Cup
+
+| Name | Start | End | ![Gold crown][GCrown] | ![Silver crown][SCrown] | ![Bronze crown][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| [osu! FumoFumo Cup cirno](OFFC/cirno) | 2024-09-15 | - | - | - | - |
```
Comment (Joehuu) [inline wiki/Tournaments/en.md:302]:
```suggestion
| [osu! FumoFumo Cup: cirno](OFFC/cirno) | 2024-09-15 | - | - | - | - |
```

File: wiki/Tournaments/zh.md [hunk 9]
```diff
@@ -261,6 +261,12 @@ outdated_translation: true
 | osu! European Tournament 3 | 2019-02-08 | 2020-03-23 | Woopak :CRAB: | Fondermanp2p | Käptn Mein Käptn |
 | [osu! European Tournament 2021](o!ET/2021) | 2021-05-15 | 2021-07-11 | opa pizdec | The Cow Fields | Team Papayapa |
 
+#### osu! FumoFumo Cup
+
+| 名称 | 开始时间 | 结束时间 | ![冠军][GCrown] | ![亚军][SCrown] | ![季军][BCrown] |
+| :-- | :-- | :-- | :-- | :-- | :-- |
+| [osu! FumoFumo Cup cirno](OFFC/cirno) | 2024-09-15 | - | - | - | - |
```
Comment (Joehuu) [inline wiki/Tournaments/zh.md:268]:
```suggestion
| [osu! FumoFumo Cup: cirno](OFFC/cirno) | 2024-09-15 | - | - | - | - |
```

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+| **Bingo!** | 诟撡𤷙𣧏𡣀𠔑𡉄𫟺㻵𧂂𨾻𠪧 | [#4](https://osu.ppy.sh/community/matches/115528531) |
+
+### Semifinals
+
+Match time: 21 September 2024 ~ 29 September 2024
+
+| Team 1 | Team 2 | Match link |
+| --: | :-- | :-- |
+| **这是什么** | Bingo! | [#5](https://osu.ppy.sh/community/matches/115660580) |
+| Pingu~Pingu~ | **沟槽的烤翅还在开战舰追我** | [#6](https://osu.ppy.sh/community/matches/115676257) |
```
Comment (Joehuu) [inline wiki/Tournaments/OFFC/cirno/en.md:219]:
```suggestion
| Pingu\~Pingu\~ | **沟槽的烤翅还在开战舰追我** | [#6](https://osu.ppy.sh/community/matches/115676257) |
```

File: wiki/Tournaments/OFFC/cirno/zh.md [hunk 11]
```diff
@@ -0,0 +1,386 @@
...
+| **Bingo!** | 诟撡𤷙𣧏𡣀𠔑𡉄𫟺㻵𧂂𨾻𠪧 | [#4](https://osu.ppy.sh/community/matches/115528531) |
+
+### 半决赛轮 (SF Round)
+
+比赛时间：2024 年 9 月 21 日 ~ 2024 年 9 月 29 日
+
+| 队伍 1 | 队伍 2 | 比赛链接 |
+| --: | :-- | :-- |
+| **这是什么** | Bingo! | [#5](https://osu.ppy.sh/community/matches/115660580) |
+| Pingu~Pingu~ | **沟槽的烤翅还在开战舰追我** | [#6](https://osu.ppy.sh/community/matches/115676257) |
```
Comment (Joehuu) [inline wiki/Tournaments/OFFC/cirno/zh.md:225]:
```suggestion
| Pingu\~Pingu\~ | **沟槽的烤翅还在开战舰追我** | [#6](https://osu.ppy.sh/community/matches/115676257) |
```

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+
+| Column / Row | 1 | 2 | 3 | 4 |
+| :-: | :-: | :-: | :-: | :-: |
+| 1 | DT3 | FM3 | HD1 | DT2 |
+| 2 | DT1 | NM4 | FM1 | HR1 |
+| 3 | HR2 | HD2 | NM2 | FM2 |
+| 4 | NM3 | FM4 | DT4 | NM1 |
+
+- NoMod
+  1. [Zektbach - Apocalypse ~dirge of swans~ [Apocalypse of the Mind]](https://osu.ppy.sh/beatmapsets/1936602#osu/4002902)
```
Comment (Joehuu) [inline wiki/Tournaments/OFFC/cirno/en.md:135]:
```suggestion
  1. [Zektbach - Apocalypse \~dirge of swans\~ [Apocalypse of the Mind]](https://osu.ppy.sh/beatmapsets/1936602#osu/4002902)
```

File: wiki/Tournaments/OFFC/cirno/zh.md [hunk 13]
```diff
@@ -0,0 +1,386 @@
...
+
+| 列/行 | 1 | 2 | 3 | 4 |
+| :-: | :-: | :-: | :-: | :-: |
+| 1 | DT3 | FM3 | HD1 | DT2 |
+| 2 | DT1 | NM4 | FM1 | HR1 |
+| 3 | HR2 | HD2 | NM2 | FM2 |
+| 4 | NM3 | FM4 | DT4 | NM1 |
+
+- NoMod
+  1. [Zektbach - Apocalypse ~dirge of swans~ [Apocalypse of the Mind]](https://osu.ppy.sh/beatmapsets/1936602#osu/4002902)
```
Comment (Joehuu) [inline wiki/Tournaments/OFFC/cirno/zh.md:141]:
```suggestion
  1. [Zektbach - Apocalypse \~dirge of swans\~ [Apocalypse of the Mind]](https://osu.ppy.sh/beatmapsets/1936602#osu/4002902)
```

File: wiki/Tournaments/OFFC/cirno/en.md [hunk 14]
```diff
@@ -0,0 +1,225 @@
...
+- [Discord server](https://discord.gg/YYprGehNxD)
+- [Matchup table](https://docs.google.com/spreadsheets/d/14ZFqYajLDRjr863hDo5n5xvGQMoDZPmNwJycgQXr3YQ/edit?gid=868349196#gid=868349196)
+- **[OFFC: cirno Main spreadsheet (Google Docs)](https://docs.google.com/spreadsheets/d/14ZFqYajLDRjr863hDo5n5xvGQMoDZPmNwJycgQXr3YQ)**
+- **[Tournament match information](https://docs.google.com/spreadsheets/d/14ZFqYajLDRjr863hDo5n5xvGQMoDZPmNwJycgQXr3YQ/edit?gid=1807200006)**
+
+## Participants
+
+| Team # | Name | Members |
+| :-: | :-: | :-- |
+| **#1** | 沟槽的烤翅还在开战舰追我 | [**Dura**](https://osu.ppy.sh/users/5185609), [ayaponzu](https://osu.ppy.sh/users/13291701), [suzuka1](https://osu.ppy.sh/users/29785618), [bad kitten](https://osu.ppy.sh/users/13875116), [NaimuTongzi](https://osu.ppy.sh/users/8883892), [chanliuko](https://osu.ppy.sh/users/32289237), [Fyu_Neru](https://osu.ppy.sh/users/6699436), [MyAngelMizuki_](https://osu.ppy.sh/users/15154497) |
```
Comment (Joehuu) [inline wiki/Tournaments/OFFC/cirno/en.md:74]: I think these should have flags near usernames. Looking at a few, some members are from different countries.

See https://osu.ppy.sh/wiki/en/Tournaments/IDC/2020#participants for example, still done even if everyone is in the same country.

File: wiki/Tournaments/OFFC/cirno/img/logo.png
Comment (Joehuu) [inline wiki/Tournaments/OFFC/cirno/img/logo.png:1]: Image is maybe a bit too big on the markdown preview. Unsure of the regulations / recommended sizes on that.

**Review by Joehuu — APPROVED** [review]
Looks good otherwise.

File: wiki/Tournaments/OFFC/cirno/img/logo.png
Comment (Joehuu) [reply wiki/Tournaments/OFFC/cirno/img/logo.png:1]: Image resize looks fine. Can add a `@2x` image (900x900) for future wiki use (when implemented) when the article is updated again. Merging for now.

**Review by Joehuu — APPROVED** [review]

**Pull request merged**